src/main/java/uk/gov/crowncommercial/dts/scale/cat/model/entity/ProjectUserMapping.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.crowncommercial.dts.scale.cat.model.entity;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+import java.time.Instant;
+
+/**
+ * JPA entity representing a mapping between a project event OCID (authority + prefix + internal ID)
+ * and Jaggaer internal event code
```

Comment:
Not sure the description is quite right - it doesn't use OCID and not mapping between OCID and event code I think? (more mapping between event and user?)